File: backend/src/test/java/com/kiwi/usersettings/UserSettingsTestFactory.java
package com.kiwi.usersettings;

public class UserSettingsTestFactory {

    public static UserSettingsDTO validUserSettingsDTO() {
        return new UserSettingsDTO(
                "finn@thehuman.com",
                70,                       
                80
        );
    }

    public static UserSettingsDTO updatedUserSettingsDTO() {
        return new UserSettingsDTO(
                "finn@thehuman.com",
                50,                       
                40
        );
    }

    public static UserSettingsDTO invalidUserSettingsDTO() {
        return new UserSettingsDTO(
                "bmolovesfootball.com",   
                150,                      
                -10                     
        );
    }
}

File: frontend/src/tests/userSettings/UserSettingsTestFactory.ts
import {UserSettingsDTO} from "../../userSettings/types/UserSettingsDTO";

export const validUserSettingsDTO: UserSettingsDTO = {
    email: 'finn@thehuman.com',
    areNotificationsEnabled: true,
    theme: 'DARK'
};

export const updateUserSettingsDTO: UserSettingsDTO = {
    email: 'finn@thehuman.com',
    areNotificationsEnabled: false,
    theme: 'LIGHT'
};

export const invalidUserSettingsDTO: UserSettingsDTO = {
    email: 'bmolovesfootball.com',
    areNotificationsEnabled: true,
    theme: 'DARK'
};